Analysis/EventFiltering/filterTables.h

```diff
@@ -17,19 +17,29 @@ namespace o2::aod
 {
 namespace filtering
 {
-DECLARE_SOA_COLUMN(H2, hasH2, bool);
-DECLARE_SOA_COLUMN(H3, hasH3, bool);
-DECLARE_SOA_COLUMN(He3, hasHe3, bool);
-DECLARE_SOA_COLUMN(He4, hasHe4, bool);
+DECLARE_SOA_COLUMN(H2, hasH2, bool);   //!
+DECLARE_SOA_COLUMN(H3, hasH3, bool);   //!
+DECLARE_SOA_COLUMN(He3, hasHe3, bool); //!
+DECLARE_SOA_COLUMN(He4, hasHe4, bool); //!
+
+// diffraction
+DECLARE_SOA_COLUMN(DG, hasDG, bool); //! Double Gap events, DG
 
 } // namespace filtering
 
-DECLARE_SOA_TABLE(NucleiFilters, "AOD", "Nuclei Filters", filtering::H2, filtering::H3, filtering::He3, filtering::He4);
+DECLARE_SOA_TABLE(NucleiFilters, "AOD", "Nuclei Filters", //!
+                  filtering::H2, filtering::H3, filtering::He3, filtering::He4);
 
-constexpr std::array<char[32], 1> AvailableFilters{"NucleiFilters"};
-constexpr std::array<char[16], 1> FilterDescriptions{"Nuclei Filters"};
+constexpr std::array<char[32], 2> AvailableFilters{"NucleiFilters", "DiffractionFilters"};
+constexpr std::array<char[16], 2> FilterDescriptions{"Nuclei Filters", "Diffraction Filters"};
```

Comment:
In your case the description should be "DiffFilters" (what you put as description of the table in line 39

Reply:
done